digitalpublicgoods/immunization-calculation-engine.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "Immunization Calculation Engine",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://cdsframework.atlassian.net/wiki/spaces/CDSF/pages/14352414/Open-Source+License"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": "N/A"
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://cdsframework.atlassian.net/wiki/spaces/ICE/pages/23494658/Documentation"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "No",
+    "checkNonPIIAccessMechanism": "Unknown",
+    "nonPIIAccessMechanism": ""
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "HIPAA (US)"
+    ],
+    "adherenceSteps": [
+      "https://cdsframework.atlassian.net/wiki/spaces/ICE/pages/23494658/Documentation"
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "HL7 VMR, FHIR, Terminology (US CDC CVX, SNOMED-CT)"
+    ],
+    "evidenceStandardSupport": [
+      "https://cdsframework.atlassian.net/wiki/spaces/ICE/pages/23494658/Documentation"
+    ],
+    "implementBestPractices": "Yes",
+    "bestPracticesList": [
+      "Agile development, consultation of relevant clinical guidelines"
+    ]
+  },
+  "doNoHarm": {
+    "preventHarm": {
+      "stepsToPreventHarm": "Yes",
+      "additionalInfoMechanismProcessesPolicies": "N/A"
+    },
+    "dataPrivacySecurity": {
+      "collectsPII": "No",
+      "typesOfDataCollected": [
+        ""
+      ],
+      "thirdPartyDataSharing": "Unknown",
```

Comment:
`Unknown` is not allowed here. It's either `Yes` or `No`, and if it's yes, they should explain in `dataSharingCircumstances`